beast/fitting/pdf2d.py

```diff
@@ -97,7 +102,7 @@ def __init__(
         for i in range(self.nbins_p1):
             for j in range(self.nbins_p2):
                 # find the indicies for the current bin
-                cur_bin_indxs, = np.where(
+                (cur_bin_indxs,) = np.where(
```

Comment:
what does parenthesis do here?

Reply:
The `black` autoformatting code adds them.  Some PEP statement about this somewhere I would assume.  Makes it explicit and easier to read that is a tuple decomposition I would assume.

Reply:
Aha!